cilium-sysdump/__main__.py

```diff
@@ -44,6 +41,8 @@ def parse_comma_sep_list(arg_string):
     parser = argparse.ArgumentParser(description='Cluster diagnosis '
                                                  'tool.')
 
+    # The "sysdump" argument is left in here for backwards compatibility.
+    parser.add_argument('sysdump', type=str, help=argparse.SUPPRESS)
```

Comment:
> This also simplifies invocation to just:
>
>   python cilium-sysdump.zip
>
> The old "python cilium-sysdump.zip sysdump" will continue to function.

This actually doesn't work for me (Python 3.8.1):
```
$ python cilium-sysdump.zip
usage: cilium-sysdump.zip [-h] [--cilium-ns CILIUM_NS] [--hubble-ns HUBBLE_NS] [--cilium-labels CILIUM_LABELS] [--hubble-labels HUBBLE_LABELS] [-v]
                          [--nodes NODES] [--since SINCE] [--size-limit SIZE_LIMIT] [--output OUTPUT] [--quick QUICK]
cilium-sysdump.zip: error: the following arguments are required: sysdump
```

Reply:
Good catch, this fixes it:
```
-    parser.add_argument('sysdump', type=str, help=argparse.SUPPRESS)
+    parser.add_argument('sysdump', type=str, nargs='?', help=argparse.SUPPRESS)
```